style.css

```diff
@@ -374,95 +471,382 @@ li {
   margin: 0 12px;
 }
 
-/* Pair programing... */
-input,
-textarea {
-  color: #3c3a39;
-  font-size: 0.9375rem;
-  font-weight: 400;
-  line-height: 24px;
-}
-
-input[type="email"] {
-  margin: 0.5625em auto 1.5em auto;
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  margin: 24px;
+  border-bottom: 1px solid #dbd8d7;
 }
 
-input::placeholder,
-textarea::placeholder {
-  font-family: "Inter", sans-serif;
-  color: #979493;
-  font-size: 0.9375rem;
-  font-weight: 400;
+.form form input {
+  font-family: "Inter", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
   line-height: 24px;
-}
-
-.contact-section {
-  background: #fff;
-  font-family: "Inter", sans-serif;
-  padding: 7.5em 1.5em 0 1.5em;
-}
-
-.contact-section-heading {
-  font-size: 2rem;
-  font-weight: 700;
-  line-height: 44px;
-  color: #172b4d;
-  margin-bottom: 3.625em;
-  text-align: center;
-}
-
-.user-info,
-.input-message {
-  width: 100%;
-}
-
-.user-info {
+  color: #979493;
+  padding: 1vw;
+  width: 95%;
   border: none;
   border-bottom: 1px solid #dbd8d7;
-  padding: 0.9375em 1em;
+  margin-top: 15px;
 }
 
-.input-message {
-  background: #fbf8f7;
+.form form textarea {
+  padding: 1vw;
   border: none;
-  padding: 1em 0 0 1em;
-}
-
-.submit-btn-container {
-  text-align: center;
+  font-family: "Inter", serif;
+  font-size: 15px;
+  background: #fbf8f7;
+  margin-top: 24px;
+  width: 96%;
 }
 
-.submit-btn-container button {
-  border: none;
+#sub-btn {
+  background: #ff6b00;
+  font-family: "Inter", serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  width: 131px;
+  padding: 12px;
+  margin: 24px 1px 154px;
 }
 
-input:focus {
-  outline: none;
+input::placeholder,
+textarea::placeholder {
+  font-family: "Inter", serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #979493;
 }
 
-.social-links {
-  border-top: 2px solid #dfe1e6;
-  padding-top: 0.9375em;
-  margin-top: 7.5em;
+textarea::placeholder {
+  color: #3c3a39;
 }
 
-.social-links-list {
+footer {
   display: flex;
-  justify-content: space-evenly;
-  width: 65%;
-  margin-inline: auto;
-}
-
-.social-link-item {
-  text-align: center;
+  align-items: center;
+  flex-direction: column;
 }
 
-.separator {
-  height: 5px;
-  width: 35%;
-  background-color: #091e42;
-  border-radius: 5px;
-  margin-inline: auto;
-  margin-top: 1.3125em;
-  margin-bottom: 0.5em;
+@media (min-width: 992px) {
+  .container {
+    background-position: 100% 25%;
+    background-image: url(./images/bgimgdesk.svg);
+    background-repeat: no-repeat;
+    height: 850px;
+    overflow: hidden;
+  }
+
+  .container .icon ul {
+    display: flex;
+    flex-direction: column;
+  }
+
+  footer .dot {
+    display: none;
+  }
```

Comment:
- [ ] Kindly make sure to stick to the design while aligning the social media links to look like the provided Figma design. :wink: 
- [ ] Make sure the horizontal line touches but ends on the screen like on the provided Figma design. :100: 

   - ### Your design

   ![image](https://user-images.githubusercontent.com/87197412/186767928-db634188-0ee7-434c-9b08-6d7dd8d07ead.png)

   - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186767792-57bc45f2-d66f-46bc-b752-a579b01cbcef.png)

Reply:
Thank you for the comment,I have Implemented it